Add comment fragment

RyanKusters opened the pull request.

RyanKusters committed:
Created Add Comment dialog Fragment and accompanying layout addcommentdialog.xml
as before added commented data manipulation for coordination.

linster commented:
Didn't I merge this before I went home?

stephenjust commented:
You merged the version that ryan closed when he didn't submit it properly... Because he had the branches backwards...

stephenjust commented:
No you didn't, I can't read.

stephenjust commented:
Looks fine. You get to merge it yourself because github can't :)

RyanKusters committed:
Resolved merge conflict on strings.xml

Merge branch 'master' of https://github.com/CMPUT301F14T14/Project into AddCommentFragment

Conflicts:
	QuestionApp/res/values/strings.xml
